### DISCUSSION TITLE
Small UI bug with hidden articles

### DISCUSSION THREAD

**Discussion by sofietoft** [GitbookIO/community#717] [0. General bugs and questions]
Hi!

So, I've just recently added our first "hidden article" to one of our docs sites.
The article is added as the only sub article to one of the top-level articles, and here's what happens:

https://github.com/user-attachments/assets/ed0da464-6a17-40db-9d49-205c02f43551

The parent page registers that a sub page has been added, and therefor adds to option to extend the list of subpages in the public docs.
The list is blank though, as the only sub article there is actually hidden 🙈 

See it in action here on the docs site: https://docs.umbraco.com/umbraco-forms
The hidden article is under the "Version specific upgrade notes".
Link to the hidden article: https://docs.umbraco.com/umbraco-forms/upgrading/version-specific/migration-ids 

Hope it makes sense!
Let me know if more info is required 🙌

**Comment by utamayo** [comment]
yep same to me
